features/subresource-integrity.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Subresource Integrity
+category: security
+bugzilla: 992096
+firefox_status: 43
+mdn_url: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity 
+spec_url: https://www.w3.org/TR/SRI/ 
+chrome_ref: 6183089948590080
+ie_ref: 6183089948590080
```

Comment:
`ie_ref` should be the name property from the IE data (https://raw.githubusercontent.com/MicrosoftEdge/Status/production/app/static/ie-status.json), which in this case would be `Subresource Integrity`.
You can also find it here: https://dev.windows.com/en-us/microsoft-edge/platform/status/subresourceintegrity